Makefile.toml

```
@@ -0,0 +1,85 @@
+[config]
+default_to_workspace = false
+
+[tasks.ignore-members]
+workspace = false
+
+[tasks.clean]
+command = "cargo"
+args = ["clean"]
+
+[tasks.build]
+dependencies = ["clean"]
+command = "cargo"
+args = ["build"]
+
+[tasks.test]
+dependencies = ["clean"]
+command = "cargo"
+args = ["test"]
+
+[tasks.format_inst]
```

If you add a feature opt-in for std you can also add this step and call it in `ci`, `ci-remote` tasks

```suggestion
[tasks.test_no_std]
dependencies = ["clean"]
command = "cargo"
args = ["test", "--no-default-features"]

[tasks.format_inst]
```

---

Added `#![no_std]` in `lib.rs`.